src/utils/middleware/setSentryUser.js

```diff
@@ -0,0 +1,15 @@
+import * as Sentry from '@sentry/node'
+import { get } from 'lodash/object'
+import {
+  CUSTOM_REQ_DATA_KEY,
+  AUTH_USER_KEY,
+} from 'src/utils/middleware/constants'
+
+const setSentryUser = (handler) => async (req, res) => {
```

Comment:
Nice job. Let's add a comment that this relies on being called after `addUserFromAuthorizationToken` middleware.